components/footer/Footer.tsx

```diff
@@ -198,39 +269,102 @@ function Footer({
   const _links = layout?.hide?.extraLinks
     ? <></>
     : <ExtraLinks content={extraLinks} />;
-  const _contacts = layout?.hide?.contacts
-    ? <></>
-    : <Contacts content={contacts} />;
 
   return (
     <footer
       class={`w-full flex flex-col pt-10 pb-[15px] gap-[15px] ${
         ColorClasses(layout)
       }`}
     >
-      <div class="lg:container mx-6 lg:mx-auto">
+      <div class="w-full">
         {(!layout?.variation || layout?.variation == "Variation 1") && (
-          <div class="flex flex-col gap-[15px] mx-[15px]">
-            <div class="flex flex-col pb-20 md:flex-row md:justify-between md:flex-wrap lg:flex-nowrap gap-8 lg:gap-12">
-              <div class="lg:mb-0 mb-6">
-                {_logo}
-                {_payments}
+          <div>
+            <Divider class="border-primary" />
+            <div class="flex flex-row justify-center pt-[45px] pb-[51px] md:pt-[65px] md:pb-[71px] md:bg-transparent bg-[#F6F6F6]">
+              {_newsletter}
+            </div>
+            <Divider class="border-primary" />
+            <div class="flex flex-col w-full flex-grow gap-5">
+              <div class="w-full flex flex-row items-center justify-between pl-[29px] pr-[25px] pt-[57px] md:pt-[45px]">
+                {footerLogos?.map((logo) => (
+                  <img
+                    src={logo.src}
+                    width={logo.width}
+                    height={logo.height}
+                    alt={logo.alt}
+                  />
+                ))}
               </div>
-              {_sectionLinks}
-              <div class="flex flex-col gap-5 lg:my-0 my-6">
-                {_contacts}
-                {_social}
+              <div class="text-center w-full pb-[19px] md:pb-[49px] pt-[10px]">
+                <Image
+                  class="md:mx-auto px-4 md:px-0"
+                  src={footerTexts?.footerTitle ?? ""}
+                  alt={footerTexts?.alt}
+                  width={514}
+                  height={38}
+                />
+                <p class="pb-9 text-[11px] font-normal text-primary pt-5">
+                  {footerTexts?.subTitle}
+                </p>
+                <div
+                  class="md:mx-auto px-4 md:px-0 md:max-w-[512px] w-full font-normal text-xs leading-[22.8px] tracking-[2%]"
```

Comment:
Nunca use demais em pixels (porcentagem ta ok), os navegadores geralmente arredondam isso, o problema é que cada navegador faz o que quer, entao prefira usar sempre o valor arredondado para que fique igual em todo o lugar
```suggestion
                  class="md:mx-auto px-4 md:px-0 md:max-w-[512px] w-full font-normal text-xs leading-[22px] tracking-[2%]"
```